### DISCUSSION TITLE
How to add blog on github.blog?

### DISCUSSION THREAD

**Discussion by Rupa-Rd** [community/community#122748] [Other Feature Feedback, Questions, & Ideas]
Hey everyone, I'm @Rupa-Rd. 

When I was going through GitHub's blog platform, I found really awesome blogs on it. I'm just wondering, it is possible to add our blogs into the GitHub blog platform (https://github.blog/). If yes, then can anyone help me with the procedure?

Thanks! 😄

**Comment by ziroziro** [comment]
I have the same issue, anyone know?

**Comment by CopperEagle** [comment]
Hey @Rupa-Rd 

As far as I know, only github staff can post on the github.blog platform. 

Any author on the platform that had a description about them was part of github. Sometimes they did not have any description on the blog platform itself but their profile linked to their github account. There, they typically stated their affiliation.

Hope this answers your question.

**Reply by Rupa-Rd** [reply]
Yeah,
I also went through some of the author's profiles just now, they are working at GitHub.

I hope they will enable the option to all users in the future. Even in the GitHub ReadMe project (https://github.com/readme), only GitHub employees are posting their articles.

Thank you! @CopperEagle

**Comment by goldieowner17** [comment]
Hello @Rupa-Rd,
Thanks for being a part of the GitHub Community, we're glad you're here!

To answer your question, yes only GitHub employees can post to github.blog.

Also, for this post, the Discover category is for conversations related to to GitHub features etc. I have moved this post to General.
